ksql-serde/src/main/java/io/confluent/ksql/serde/KsqlTopicSerDe.java

```diff
@@ -39,4 +43,10 @@ protected KsqlTopicSerDe(final DataSource.DataSourceSerDe serDe) {
                                                        KsqlConfig ksqlConfig,
                                                        boolean isInternal,
                                                        SchemaRegistryClient schemaRegistryClient);
+
```

Comment:
Why do we keep this one after adding the following method?

Reply:
It's used by tests.